ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -30,7 +30,7 @@ let prototyped_of_field = function
   | "VTPM", "persistence_backend" ->
       Some "22.26.0"
   | "host", "numa_affinity_policy" ->
-      Some "23.32.0-next"
+      Some "24.2.0"
```

Comment:
Ah, could you change these here and below to 24.0.0 please?

Reply:
How about removing these changes entirely (I usually don't include them)?

Reply:
It needs to be done eventually...